x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/components/enablement_modal.tsx

```diff
@@ -131,13 +147,20 @@ export const EntityStoreEnablementModal: React.FC<EntityStoreEnablementModalProp
       </EuiModalBody>
 
       <EuiModalFooter>
-        <EuiButtonEmpty onClick={() => toggle(false)}>{'Cancel'}</EuiButtonEmpty>
-        <EuiButton onClick={enableStore(enablements)} fill>
-          <FormattedMessage
-            id="xpack.securitySolution.entityAnalytics.enablements.modal.enable"
-            defaultMessage="Enable"
-          />
-        </EuiButton>
+        <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+          <EuiFlexItem>{!enablementOptions && proceedWarning}</EuiFlexItem>
```

Comment:
nit? When `enablementOptions` is false, do we render an empty div?

Maybe this could work:
```suggestion
         {enablementOptions ?  <EuiFlexItem> <EuiCallOut...  : null}
```